.travis.yml

```diff
@@ -46,6 +46,7 @@ stages:
   - name: test
   - name: experiment
     if: type = push
+  - name: bundlesize
```

Comment:
I think it's better to have the bundle size job in the `test` stage - as it currently stands, if any of the test or experiment jobs fail on a push build, this commit won't have generated bundle size data and will cause subsequent PRs to fail

/cc @rsimha for second opinion

Reply:
+1. This can be part of the test stage.

Reply:
done. does it matter what order it appears in the list?